tox.ini

```diff
@@ -12,10 +12,12 @@ setenv =
     PYTEST_COMMAND = pytest --pyargs plasmapy --durations=25 -n=auto --dist=loadfile
 extras = tests
 deps =
-    numpydev: git+https://github.com/numpy/numpy
     astropydev: git+https://github.com/astropy/astropy
-    xarraydev: git+https://github.com/pydata/xarray
+    matplotlibdev: git+https://github.com/matplotlib/matplotlib
+    numpydev: git+https://github.com/numpy/numpy
+    scipydev: git+https://github.com/matplotlib/matplotlib
```

Comment:
Okay, first up that last one points to matplotlib rather than scipy :) Secondly, and this is something I've been meaning to improve for a while - this way of installing these three packages means we're building huge binary blobs from scratch, wasting time and energy.

Instead, we can leverage prebuilt wheels as per [sklearn docs](https://scikit-learn.org/stable/developers/advanced_installation.html#installing-nightly-builds): something like
```bash
pip install --pre --extra-index https://pypi.anaconda.org/scipy-wheels-nightly/simple scikit-learn
````
should do the trick.

In fact, we can replace all three of these, since they're listed under https://anaconda.org/scipy-wheels-nightly/repo?type=pypi&label=main.

We just need to figure out whether `tox deps` will take this kind of argument. Checking...

Reply:
... for some reason I saw matplotlib where it wasn't there. I've pushed a commit to try this out.

Reply:
That's a really good idea!  Being able to run these tests quickly if definitely helpful, especially since we'd need to trigger runs occasionally to see if things get fixed.